autonomi/src/client/data_types/pointer.rs

```diff
@@ -29,21 +40,48 @@ pub enum PointerError {
     InvalidQuote,
     #[error("Pointer already exists at this address: {0:?}")]
     PointerAlreadyExists(PointerAddress),
+    #[error("Pointer cannot be updated as it does not exist, please create it first or wait for it to be created")]
+    CannotUpdateNewPointer,
 }
 
 impl Client {
     /// Get a pointer from the network
     pub async fn pointer_get(&self, address: PointerAddress) -> Result<Pointer, PointerError> {
+        info!("Getting pointer: {address:?}");
+
         let key = NetworkAddress::from_pointer_address(address).to_record_key();
-        let record = self.network.get_local_record(&key).await?;
+        debug!("Fetching pointer from network at: {key:?}");
```

Comment:
this is somehow `duplicated` with the log at the beginning of the function?
NetworkAddress debug format already show the key is about a Pointer

Reply:
Ooh exactly!